src/main/java/no/nav/pto/veilarbportefolje/persononinfo/PdlRepository.java

```diff
@@ -0,0 +1,77 @@
+package no.nav.pto.veilarbportefolje.persononinfo;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import no.nav.common.types.identer.AktorId;
+import no.nav.pto.veilarbportefolje.persononinfo.PdlResponses.PDLIdent;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import static no.nav.pto.veilarbportefolje.postgres.PostgresUtils.queryForObjectOrNull;
+
+@Slf4j
+@Repository
+@RequiredArgsConstructor
+public class PdlRepository {
+    @Qualifier("PostgresJdbc")
+    private final JdbcTemplate db;
+
+    @Transactional
+    public void upsertIdenter(List<PDLIdent> pdlIdenter) {
```

Comment:
Bør vi her vurdere å finne alle lokale identer for `pdlIdenter`, slette alle rader for resulterende lokale identer, og deretter insert? Det er kanskje unødvendig mye arbeid, men da kunne man brukt samme logikk ved start oppfølging som når man får endring inn fra Kafka-topic'en.

Reply:
Jepp dette må gjøres, ellers kan det nok fort oppstå bugs